packages/jest-cli/src/__tests__/generate_empty_coverage.test.js

```diff
@@ -4,14 +4,13 @@
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
- *
- * @emails oncall+jsinfra
  */
 'use strict';
 
 const os = require('os');
 
 const generateEmptyCoverage = require('../generate_empty_coverage');
+const {makeGlobalConfig, makeProjectConfig} = require('../../../../test_utils');
```

Comment:
we had a `@providesModule `for `skipOnWindows`, but it wasn't typed with flow because of the custom resolution logic. I'm not sure how to make flow understand the mapping in a reasonable way, but i know it's possible, cause we use it at fb :)
At the same time we can just maintain two mappings in sync (for jest and for flow)
That should probably be a separate issue that will do that for other shared modules as well